### ISSUE TITLE
When only one partition of the shomate equation is needed the code doesn't work and gives error: TypeError: 'float' object is not subscriptable

### ISSUE THREAD

**Issue by Kosteand** [NickCodesStuf/RocketEngineEngineer#1]
When only one partition of the shomate equation is needed the code doesn't work and gives error: TypeError: 'float' object is not subscriptable. This is most likely because the code excepts a 2d array but gets a list.

**Comment by NickCodesStuf** [comment]
Just put an extra set of square brackets [] around it. 
My functions are made to iterate through a 2 dimensional array, because it can support multiple partitions of data without having to break it up. it is fine if one dimension only has 1 element to it, but it is a niche use case.

**Comment by NickCodesStuf** [comment]
TLDR
instead of 
`[A, B, C. D, E. min, max]`
write
`[[A, B, C, D, E, min, max]]`
